includes/class-wc-structured-data.php

```diff
@@ -91,15 +91,22 @@ public function get_structured_data( $types ) {
 
 		// Wrap the multiple values of each type inside a graph... Then add context to each type.
 		foreach ( $data as $type => $value ) {
-			$data[ $type ] = count( $value ) > 1 ? array( '@graph' => $value ) : $value[0];
-			$data[ $type ] = apply_filters( 'woocommerce_structured_data_context', array( '@context' => 'https://schema.org/' ), $data, $type, $value ) + $data[ $type ];
+			if ( 1 < count( $value ) ) {
+				$data[ $type ] = apply_filters( 'woocommerce_structured_data_context', array( '@context' => 'https://schema.org/' ), $data, $type, $value ) + array( '@graph' => $value );
+			} else {
+				$data[ $type ] = $value[0];
+			}
 		}
```

Comment:
@mikejolley, I think @saulfautley is right about the `@context`. This is putting item level `@context` only if there are multiple `$value`. Shouldn't we add `@context` to single `$value` also ?